common/secret/token.ts

```diff
@@ -158,6 +158,13 @@ export async function loginToken(token: string): Promise<User | never> {
 }
 
 export async function verifyEmail(user: User) {
+    const { id: notificationID } = await prisma.notification.findFirst({
+        where: { onActions: { has: 'user-verify-email' } },
```

Comment:
Could use move the string to an constant of type ActionID? It will ensure that there will never be a typo in the string or that we have to update this part when `user-verify-email` is removed.

Reply:
@dhenkel92 Does it even make sense to look for the concrete notification here? `verifyEmail` creates the concrete notification for `user-verify-email` for the first time.